src/components/Sidebar/Participants/ParticipantsTab.vue

```diff
@@ -0,0 +1,216 @@
+				:items="addableUsers"
+				@refreshCurrentParticipants="getParticipants" />
+			<Hint v-else-if="contactsLoading" :hint="t('spreed', 'Loading')" />
+			<Hint v-else :hint="t('spreed', 'No search results')" />
+
+			<Caption
+				:title="t('spreed', 'Add groups')" />
+			<ParticipantsList
+				v-if="addableGroups.length !== 0"
+				:items="addableGroups"
+				@refreshCurrentParticipants="getParticipants" />
+			<Hint v-else-if="contactsLoading" :hint="t('spreed', 'Loading')" />
+			<Hint v-else :hint="t('spreed', 'No search results')" />
+		</template>
+	</div>
+</template>
+
+<script>
+import Caption from '../../Caption'
+import CurrentParticipants from './CurrentParticipants/CurrentParticipants'
+import Hint from '../../Hint'
+import ParticipantsList from './ParticipantsList/ParticipantsList'
+import SearchBox from '../../SearchBox'
+import debounce from 'debounce'
+import { EventBus } from '../../../services/EventBus'
+import { CONVERSATION, WEBINAR } from '../../../constants'
+import { searchPossibleConversations } from '../../../services/conversationsService'
+import { fetchParticipants } from '../../../services/participantsService'
+
+export default {
+	name: 'ParticipantsTab',
+	components: {
+		CurrentParticipants,
+		SearchBox,
+		Caption,
+		Hint,
+		ParticipantsList,
+	},
+
+	props: {
+		displaySearchBox: {
+			type: Boolean,
+			required: true,
+		},
+	},
+
+	data() {
+		return {
+			searchText: '',
+			searchResults: [],
+			contactsLoading: false,
+		}
+	},
+
+	computed: {
+		show() {
+			return this.$store.getters.getSidebarStatus
+		},
+		opened() {
+			return !!this.token && this.show
+		},
+		token() {
+			return this.$route.params.token
+		},
+		conversation() {
+			if (this.$store.getters.conversations[this.token]) {
+				return this.$store.getters.conversations[this.token]
+			}
+			return {
+				token: '',
+				displayName: '',
+				isFavorite: false,
+				type: CONVERSATION.TYPE.PUBLIC,
+				lobbyState: WEBINAR.LOBBY.NONE,
+			}
+		},
+		isSearching() {
+			return this.searchText !== ''
+		},
+		addableUsers() {
+			if (this.searchResults !== []) {
+				const searchResultUsers = this.searchResults.filter(item => item.source === 'users')
+				const participants = this.$store.getters.participantsList(this.token)
+				return searchResultUsers.filter(user => {
+					let addable = true
+					for (const participant of participants) {
+						if (user.id === participant.userId) {
+							addable = false
+							break
+						}
+					}
+					return addable
+				})
+			}
+			return []
+		},
+		addableGroups() {
+			if (this.searchResults !== []) {
+				return this.searchResults.filter((item) => item.source === 'groups')
+			}
+			return []
+		},
+	},
+
+	beforeMount() {
+		this.getParticipants()
+		/**
+		 * If the route changes, the search filter is reset and we get participants again
+		 */
+		EventBus.$on('routeChange', () => {
+			this.searchText = ''
+			this.$nextTick(() => {
+				this.getParticipants()
```

Comment:
If the data is properly updated with reactivity, you should not need to wait for next tick?
You can already fetch the participants, no?